packages/sdk/src/actions/simulateDepositTx.ts

```diff
@@ -120,3 +98,31 @@ export async function simulateDepositTx(params: SimulateDepositTxParams) {
 
   return result as unknown as SimulateContractReturnType;
 }
+
+export async function getMaxFillDeadline(
```

Comment:
NIT: Is `maxFillDeadline` the correct naming here? The term deadline already implies sort of a max. value. Also if explicitly set by the caller, this value can be higher than the default value we calculate here

Reply:
So maybe just `getDefaultFillDeadline` or `getFillDeadline` is more appropiate

Reply:
my bad you are right